pkg/sql/sem/tree/schedule.go

```diff
@@ -38,6 +39,11 @@ func (node *ScheduledBackup) Format(ctx *FmtCtx) {
 		ctx.FormatNode(node.ScheduleLabel)
 	}
 
+	if node.IfNotExists ==true{
+		ctx.WriteString("IF NOT EXISTS")
+		//ctx.FormatNode(node.IfNotExists) ??
```

Comment:
What you have is correct! We can only recurse using `ctx.FormatNode` if the parameter is another tree node. In our case IfNotExists is a bool (which is not a tree node) so we explicitly write the string as you have.

Reply:
done